README.md

```diff
@@ -12,7 +12,7 @@ Appium Inspector is basically just an Appium client (like WebdriverIO, Appium's
 
 This version of Appium Inspector is designed to work with Appium 2.0 as a default. So if you are migrating from Appium Desktop (which is designed to work with Appium 1.x as a default), you need to be aware of some changes:
 
-* The default remote server path has changed from `/wd/hub` to `/` to reflect Appium 2.0's default server path. If you're using Appium Inspector with an Appium 1.x server, you'll likely need to update the path information in the New Session form back to `/wd/hub`.
+- The default remote server path has changed from `/wd/hub` to `/` to reflect Appium 2.0's default server path. If you're using Appium Inspector with an Appium 1.x server, you'll likely need to update the path information in the New Session form back to `/wd/hub`.
```

Comment:
I feel like this may be premature?

also... I'd hope that appium would respond with a 404 mentioning that the default base path changed. guessing it doesn't. @jlipps would that be a good idea?

Reply:
yeah that would probably be a good idea. and it may have been premature, but we made the call to have the inspector's defaults work with appium 2's defaults, so changing it back would cause even more confusion.